### ISSUE TITLE
Training Issue

### ISSUE THREAD

**Issue by xinguohe** [hxwork/H2ONet_Pytorch#4]
Thank you for your interesting project. 
But when I run the experiment on the DEX-YCB dataset after several iterations, the loss will be NAN and can not be trained.

**Comment by hxwork** [comment]
Hi,

Sorry for the confusion. We adopt a two-stage training to avoid this issue. Specifically, we do not use the losses related to the hand orientation at the first stage and the initial learning rate is set to 1e-3. In the second stage, we use all losses, and the initial learning rate is set to 1e-4. With this configuration, the NaN issue will disappear most time. If it occurs, please try to simply run the experiment again.

**Comment by xinguohe** [comment]
Thank you so much for your kind reply. Could you please provide the detailed configuration of the two-stage training strategy? It would be greatly appreciated.

**Comment by hxwork** [comment]
Actually, we have provided the reference configuration and you can easily modify it for this two-stage training in two steps:
1. train a model without applying the losses related to the hand orientation. Only the hand shape at the canonical pose is supervised in the first stage. For other settings please refer to the reference configuration. 
2. load the model weights in step 1 and change the initial learning rate to 1e-4. Use all loss functions in this stage. Other settings are unchanged.
 
You can follow these two steps to train the model, and I will release the code in this month.

**Comment by xinguohe** [comment]
Thank you very much for your reply.
